integration/test_ft_alias_cluster.py

```
@@ -0,0 +1,319 @@
+        ) == b"OK"
+
+        # Poll until alias search returns results on all primaries.
+        def _alias_search_ready():
+            for node in self._all_primaries():
+                try:
+                    result = node.execute_command(
+                        "FT.SEARCH", ALIAS_NAME, "@category:{books}"
+                    )
+                    if result[0] < 1:
+                        return False
+                except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)